aix/SPECS/3.11.0/wazuh-agent-3.11.0-aix.spec

```diff
@@ -0,0 +1,292 @@
+%dir %attr(770,ossec,ossec) %{_localstatedir}/ossec/etc
+%attr(640,root,ossec) %config(noreplace) %{_localstatedir}/ossec/etc/client.keys
+%attr(640,root,ossec) %{_localstatedir}/ossec/etc/internal_options*
+%attr(640,root,ossec) %config(noreplace) %{_localstatedir}/ossec/etc/local_internal_options.conf
+%attr(660,root,ossec) %config(noreplace) %{_localstatedir}/ossec/etc/ossec.conf
+%{_localstatedir}/ossec/etc/ossec-init.conf
+%attr(640,root,ossec) %{_localstatedir}/ossec/etc/wpk_root.pem
+%dir %attr(770,root,ossec) %{_localstatedir}/ossec/etc/shared
+%attr(660,root,ossec) %config(missingok,noreplace) %{_localstatedir}/ossec/etc/shared/*
+%dir %attr(750,root,system) %{_localstatedir}/ossec/lib
+%dir %attr(770,ossec,ossec) %{_localstatedir}/ossec/logs
+%attr(660,ossec,ossec) %ghost %{_localstatedir}/ossec/logs/active-responses.log
+%attr(660,root,ossec) %ghost %{_localstatedir}/ossec/logs/ossec.log
+%attr(660,root,ossec) %ghost %{_localstatedir}/ossec/logs/ossec.json
+%dir %attr(750,ossec,ossec) %{_localstatedir}/ossec/logs/ossec
+%dir %attr(750,root,ossec) %{_localstatedir}/ossec/queue
+%dir %attr(770,ossec,ossec) %{_localstatedir}/ossec/queue/ossec
+%dir %attr(750,ossec,ossec) %{_localstatedir}/ossec/queue/diff
+%dir %attr(770,ossec,ossec) %{_localstatedir}/ossec/queue/alerts
+%dir %attr(750,ossec,ossec) %{_localstatedir}/ossec/queue/rids
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/ruleset/sca
+%attr(640, root, ossec) %{_localstatedir}/ossec/ruleset/sca/*
+%dir %attr(1750,root,ossec) %{_localstatedir}/ossec/tmp
+%attr(750,root,system) %config(missingok) %{_localstatedir}/ossec/tmp/add_localfiles.sh
+%attr(750,root,system) %config(missingok) %{_localstatedir}/ossec/tmp/gen_ossec.sh
+%dir %attr(1750,root,ossec) %config(missingok) %{_localstatedir}/ossec/tmp/etc/templates
+%dir %attr(1750,root,ossec) %config(missingok) %{_localstatedir}/ossec/tmp/etc/templates/config
+%dir %attr(1750,root,ossec) %config(missingok) %{_localstatedir}/ossec/tmp/etc/templates/config/generic
+%attr(750,root,system) %config(missingok) %{_localstatedir}/ossec/tmp/etc/templates/config/generic/*.template
+%dir %attr(1750,root,ossec) %config(missingok) /var/ossec/tmp/etc/templates/config/generic/localfile-logs
+%attr(750,root,system) %config(missingok) /var/ossec/tmp/etc/templates/config/generic/localfile-logs/*.template
+%attr(750,root,system) %config(missingok) %{_localstatedir}/ossec/tmp/src/*
+%dir %attr(750,root,ossec) %{_localstatedir}/ossec/var
+%dir %attr(770,root,ossec) %{_localstatedir}/ossec/var/incoming
+%dir %attr(770,root,ossec) %{_localstatedir}/ossec/var/run
+%dir %attr(770,root,ossec) %{_localstatedir}/ossec/var/upgrade
+%dir %attr(770,root,ossec) %{_localstatedir}/ossec/var/wodles
+%dir %attr(750,root,ossec) %{_localstatedir}/ossec/wodles
+%dir %attr(750,root,ossec) %{_localstatedir}/ossec/wodles/aws
+%attr(750,root,ossec) %{_localstatedir}/ossec/wodles/aws/*
+
+
+%changelog
+* Mon Jun 11 2019 support <support@wazuh.com> - 3.10.0
```

Comment:
The changelog line is missing.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/254/commits/bd9469d9104aaced5b40e1a09a527bbe1cefa707